platforms/android/library/src/androidTest/java/io/element/android/wysiwyg/test/EditorEditTextInputTests.kt

```diff
@@ -308,6 +311,31 @@ class EditorEditTextInputTests {
         }
     }
 
+    @Test
+    @Ignore("Failing in CI environment")
+    fun testTextWatcher() {
```

Comment:
This test is failing only in the CI environment and I haven't managed to work out why.

@jmartinesp, I'll leave the PR open while I'm away in case you have any ideas how to fix this. Otherwise feel free to merge it as-is and I'll take a look when I'm back.